src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1218,30 +1220,34 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h tl_md, uct_worker_h wor
     }
 
     init_attr.qp_type     = UCT_IB_QPT_DCI;
-    init_attr.flags       = UCT_IB_CQ_IGNORE_OVERRUN;
+    init_attr.flags       = UCT_IB_CQ_IGNORE_OVERRUN |
+                            UCT_IB_TX_OPS_PER_PATH;
```

Comment:
instead of passing UCT_IB_TX_OPS_PER_PATH, can't we just update the calculation in line 1231-1233 below?

Reply:
how can we update it?
ib_num_path will be initialized only in IB iface constructor, while we have to pass some CQ length to it as a parameter

Reply:
i see, maybe worth to comment this reason when passing this flag in dc_mlx5 code?

Reply:
ok